config/config.yaml

```diff
@@ -0,0 +1,73 @@
+# netbox device attributes that should be used as filters for devices query.
+# these translate directly into the return fields from the netbox json API.
+# If no filter is specified, by default only devices with Active (1) status are pulled.
+filters:
+  status: ["active", "staged", "offline"]
+#  manufacturer: ['juniper']
+#  custom_fields.field1: "foo"
+
+# list of tags to filter devices in Netbox. Only devices that have this tag will be 
+# synchronized to NetSpyGlass. Default is None, which causes all devices present in Netbox to be synchronized.
+whitelist:
+#  - tag1
+#  - tag2
+
+# list of tags to blacklist from netbox. Devices that have this tag will not be synchronized to NetSpyGlass.
+# Blacklist filter is applied after the whitelist (if any). Default is None, which turns this off and causes all
+# devices that pass whitelist will be synchronized
+blacklist:
+#  - tagq
+
+# list of devices in nsg excluding from synchronization 
+nsg_blacklist:
+  name:
+#    - nsg device name
+  deviceId:
+#    - nsg device id
+  address:
+#    - nsg device ip
+
+# Mapping device name -> snmp channels. Device name can be in regex format
+default_channels:
+#  device_name:
+#    - v1public
+#  aaa.*|bbb-bb01|ccc-mt01:  # device names regex
+#    - v1public
+  "*":
+    - v2public
+
+#domain: your.domain.com
+
+# List of external nsg tags with corresponding netbox properties
+nsgTags:
```

Comment:
please add a few lines to document this part of the configuration.

Reply:
added